pkgs/development/tools/sqlint/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, bundlerApp, cacert }:
+
+bundlerApp {
+  pname = "sqlint";
+  gemdir = ./.;
+  exes = [ "sqlint" ];
+
+  gemConfig = {
+    pg_query = attrs: {
+      NIX_SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
What is this for?

Reply:
The package tries to download source code at runtime: https://github.com/lfittl/pg_query/blob/de2c9d1f61d7be1d2ba8be24bc495221cbc626a1/ext/pg_query/extconf.rb#L13
This will not work on hydra since we disallow network access in our build sandbox. However it is possible to pre-download the source with fetchurl instead.

Reply:
Would you recommend I create a separate package for pg_query, or should I just keep it all in this package?

Reply:
Usually I would recommend to put it into https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/ruby-modules/gem-config/default.nix
However in this case here it is tricky because it seems to require a specific version of libpg_query.
In this case better to keep the override for pg_query in this file because the version for it is fixed until sqlint receives an upgrade.

Reply:
Getting the `libpg_query` tarball is easy enough:

```
libpgQuerySrc = fetchurl {
  url = https://codeload.github.com/lfittl/libpg_query/tar.gz/10-1.0.1;
  sha256 = "0m5jv134hgw2vcfkqlnw80fr3wmrdvgrvk1ndcx9s44bzi5nsp47";
};
```

But I can't seem to copy it to the path corresponding to `"#{workdir}/libpg_query.tar.gz"` before the build starts.